teaser/data/output/modelicatemplate/AixLib/AixLib_Multizone

```diff
@@ -0,0 +1,377 @@
+      points={{-62,40},{-32,40},{-32,6},{34,6}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(weaBus.TDryBul, replicatorTemperatureVentilation.u) annotation (Line(
+      points={{-51,6},{-44,6},{-44,-12},{-37,-12},{-37,-13}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}}));
+  connect(replicatorTemperatureVentilation.y, multizone.ventTemp) annotation (
+      Line(points={{-25.5,-13},{-20,-13},{-20,3.8},{33,3.8}}, color={0,0,127}));
+  connect(tableInternalGains.y, multizone.intGains)
+    annotation (Line(points={{55.2,-34},{48,-34},{48,-9}}, color={0,0,127}));
+  connect(tableInfiltrationRate.y, multizone.ventRate) annotation (Line(points={{
+          -23.4,-32},{0,-32},{0,1},{33,1}}, color={0,0,127}));
+
+  annotation (experiment(
+      StopTime=${modelica_info.runtime_simulation},
+      Interval=${modelica_info.interval_output},
+      __Dymola_Algorithm="${modelica_info.current_solver}"),
+      __Dymola_experimentSetupOutput(events=${get_true_false(modelica_info
+      .equidistant_output)}),
+    Icon(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,100}}),
+        graphics={
+        Line(points={{80,-82}}, color={28,108,200}),
+        Rectangle(
+          extent={{-80,20},{80,-80}},
+          lineColor={0,0,0},
+          lineThickness=0.5),
+        Line(
+          points={{-80,20},{0,100},{80,20}},
+          color={0,0,0},
+          thickness=0.5),
+        Text(
+          extent={{-52,-10},{62,-48}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid,
+          textString="TB")}));
+end ${bldg.name};
+
+% elif model == "MultizoneEquipped":
+  AixLib.ThermalZones.ReducedOrder.Multizone.MultizoneEquipped multizone(
+    redeclare package Medium = Modelica.Media.Air.SimpleAir,
+    buildingID=${get_id(bldg.building_id)},
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T_start = ${bldg.thermal_zones[0].t_inside},
+    VAir = ${bldg.volume},
+    ABuilding=${bldg.net_leased_area},
+    ASurTot=${get_total_surface_area(bldg)},
+    numZones = ${get_n(bldg.thermal_zones)},
+    zoneParam = {
+      %for i in bldg.thermal_zones:
+	  %if i == bldg.thermal_zones[-1]:
+	   ${bldg.name}_DataBase.${bldg.name}_${i.name.replace(" ", "")}()
+	  %else:
+	   ${bldg.name}_DataBase.${bldg.name}_${i.name.replace(" ", "")}(),
+	  %endif
+      %endfor
+      },
+% if bldg.central_ahu:
+	heatAHU=${get_true_false(bldg.central_ahu.heating)},
+    coolAHU=${get_true_false(bldg.central_ahu.cooling)},
+    dehuAHU=${get_true_false(bldg.central_ahu.dehumidification)},
+    huAHU=${get_true_false(bldg.central_ahu.humidification)},
+    BPFDehuAHU=${bldg.central_ahu.by_pass_dehumidification},
+    HRS=${get_true_false(bldg.central_ahu.heat_recovery)},
+    sampleRateAHU=1800,
```

Comment:
I think we should include parameters to the CentralAHU class instead of adding random default values to template
`sampleRateAHU`
`effFanAHU_sup`
`effFanAHU_eta`
`dpAHU_sup`
`dpAHU_eta`